src/Symfony/Component/HttpKernel/Attribute/Cache.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Attribute;
+
+/**
+ * Describes the default HTTP cache headers on controllers.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+#[\Attribute(\Attribute::TARGET_CLASS | \Attribute::TARGET_METHOD | \Attribute::TARGET_FUNCTION)]
+class Cache
```

Comment:
```suggestion
final class Cache
```

as subclasses are not supported atm

Reply:
updated